docs/Makefile

```diff
@@ -2,7 +2,7 @@ BUILDDIR     = build
 LINKCHECKDIR = $(BUILDDIR)/linkcheck
 SOURCEDIR    = .
 SPHINXBUILD  = sphinx-build
-SPHINXOPTS   = -a -n
+SPHINXOPTS   = -a -n -W
```

Comment:
I'm not sure if we'll be able to keep this and if it actually does what I hope, but it's supposed to upgrade warnings in the docs build to errors, meaning that, in particular, a warning in the link-check step of `make docs` should, hopefully, stop before doing the doc-build step. I've noticed during development that some warnings/errors are flagged during the link-check step that then are repeated in the doc-build step. If there's a problem, we shouldn't see it in the output twice: We should stop as soon as there's an issue. We'll see if this helps, and if it's not the case that there are not warnings that we want to ignore, such that they cannot be treated as errors.